src/main/resources/AntwoordenModel_v2_2_3.csv

```diff
@@ -1,5 +1,6 @@
 v0101,v0102,v0103,v0103A,v0103B,v0104,v0201,v02A01,v02A02,v02A03,v02B01,v0301,v0302,v04A02,V04A03,V04B01,Exception,route,soortGezag,gezag_ouder1,gezag_ouder2,gezag_niet_ouder1,gezag_niet_ouder2,Index,Uitleg
 ,,,,,,,,,,,,,,,,,0,N,Nee,Nee,Nee,Nee,0,Gezag is niet te bepalen. Neem contact op met RvIG via info@RvIG onder vermelding van BRP API gezag met de volgende foutcode: %s
+,,,,,,,,,,,,,,,,,0m,N,Nee,Nee,Nee,Nee,0,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: %s
```

Comment:
Ik zou hier route 1e verwachten. 1 omdat het bezig was met vraag 1 en het daar fout is gegaan. `m` is gebruikt in `2m` voor meerderjarig en alle andere "missende gegeven" toelichtingen eindigen ook op een e.

Reply:
resolved